assets/base.css

```diff
@@ -2432,6 +2454,12 @@ input[type='checkbox'] {
   background: rgba(var(--color-foreground), 0.5);
 }
 
+@media screen and (max-width: 989px) {
+  menu-drawer ~ .header__icons .header__icon--account {
+    display: none;
+  }
+}
+
 /* Search */
 menu-drawer + .header__search {
```

Comment:
I noticed something when the header is set to the `top center` layout. The search icon needs a negative margin like the icon to be vertically aligned with the rest of the content: 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/14-24-5u5jr-rgs16.png)
</details>

But that's not something touched in this PR anyway. Ill create an issue for it 👍 
Issue -> https://github.com/Shopify/dawn/issues/2300